package.json

```diff
@@ -77,32 +80,9 @@
     ]
   },
   "main": "src/index.js",
-  "semistandard": {
-    "globals": [
-      "alert",
-      "cartodb",
-      "cdb",
-      "jasmine",
-      "describe",
-      "xdescribe",
-      "beforeEach",
-      "afterEach",
-      "spyOn",
-      "it",
-      "xit",
-      "expect"
-    ],
-    "ignore": [
-      "/*.js",
-      "/dist",
-      "/examples",
-      "/grunt-tasks",
-      "/styleguide",
-      "/themes",
-      "/vendor"
-    ]
-  },
   "scripts": {
-    "test": "grunt test"
+    "test": "grunt test",
+    "lint": "./node_modules/.bin/eslint .",
```

Comment:
Since [npm already resolves it](https://docs.npmjs.com/misc/scripts#path)  I think we can use eslint without specifying the path

Reply:
Great!